src/py/run_program.rs

```diff
@@ -36,9 +54,14 @@ pub fn serialize_and_run_program(
     quote_kw: u8,
     apply_kw: u8,
     max_cost: u32,
+    flags: u32,
```

Comment:
I'd like to replace `flags` with an explicit reference to the function to call when we get an unknown operators, but we can do that later since it requires more complicated bridging logic mess.